modules/rhpam-apb/roles/deploy-amq/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: "[AMQ] check requirements"
+  include_role:
+    name: check-image-requirements
+  vars:
+    image_namespace: "{{ amq_image_namespace }}"
+    image_name: "{{ amq_image_name }}"
+    image_tag: "{{ amq_image_tag }}"
+  when: state == "present"
+
+- name: "[AMQ] Configure AMQ queues"
+  set_fact:
+    kieserver_jms_amq_queues: "{{ kieserver_jms_queue_executor }},{{ kieserver_jms_queue_request }},{{ kieserver_jms_queue_response}},{{ kieserver_jms_queue_signal }},{{ kieserver_jms_queue_audit }}"
+  when:
+    apb_kieserver_jms_amq_queues|trim == "" or
```

Comment:
`kieserver_jms_amq_queues`  should have the value passed in when calling the role from kieserver role. Try not to use apb_* vars at this point. Maybe you can set this default value using the `default` filter in kieserver/defaults/main.yml file

Reply:
I tried to add the default queues names on kieserver/defaults/main.yml but didn't work very well thus  I created a task for setting it.